members/TXL000379.yaml

```diff
@@ -27,13 +27,21 @@ contact_form:
           required: true
         - name: state
           selector: "input[name='state']"
-          value: $ADDRESS_STATE_POSTAL_ABBREV
+          value: TX
```

Comment:
Just curious, why did you make this change?

Reply:
No particular strong reason. When testing I changed `$ADDRESS_STATE_POSTAL_ABBREV` for `TX` as it was a bit simpler. Thought about copying that to the other yamls as well. 

question: would there be an issue if supporter info has `Texas`, but the yaml has `TX` harcoded?